app/views/discovered_hosts/welcome.html.erb

```diff
@@ -0,0 +1,9 @@
+<% content_for(:title, _("DiscoveredHosts")) %>
+<div class="blank-slate-pf">
+  <div class="blank-slate-pf-icon">
+    <%= icon_text("gears", "", :kind => "fa") %>
+  </div>
+  <h1><%= _('Discovered Hosts') %></h1>
+  <p><%= _("You don't have any visible discovered hosts. This plugin enables Foreman to do automatic bare-metal discovery of unknown nodes on the provisioning network.") %></p>
```

Comment:
Can you reword and split it to i18n keys:

    <p><%= _("No discovered hosts found in this context.") %>
    <%= _("This page shows discovered bare-metal or virtual nodes waiting to be provisioned.") %></p>